<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1309</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D Surface</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/medical.png</normaloff>../../../Downloads/medical.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QVTKWidget" name="verticalWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QVTKWidget" name="verticalWidget_3" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3"/>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QVTKWidget" name="verticalWidget_4" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4"/>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QVTKWidget" name="verticalWidget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2"/>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>&amp;Import</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="volumeSelectionComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Select Volume</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bone</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Compact Bone (Adult)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Compact Bone (Child)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Enamel (Adult)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Enamel (Child)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fat Tissue (Adult)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fat Tissue (Child)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Muscle Tissue (Adult)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Muscle Tissue (Child)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Skin Tissue (Adult)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Skin Tissue (Child)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Soft Tissue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spongial Bone (Adult)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spongial Bone (Child)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="overwriteCheckbox">
        <property name="text">
         <string>Overwrite Previous Volume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generateVolumeButton">
        <property name="text">
         <string>Generate Volume</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>148</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="clipCheckBox">
        <property name="text">
         <string>Show clipping plane</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>68</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="generateFinalSurface">
        <property name="text">
         <string>&amp;Generate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>generateFinalSurface</zorder>
   <zorder>openButton</zorder>
   <zorder>clipCheckBox</zorder>
   <zorder>volumeSelectionComboBox</zorder>
   <zorder>generateVolumeButton</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>overwriteCheckbox</zorder>
   <zorder>checkBox_2</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>verticalSpacer_2</zorder>
   <zorder>verticalSpacer_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1309</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global">QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
